docs/guides/server/bootstrap-admin-recovery.adoc

```diff
@@ -0,0 +1,107 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Admin bootstrap and recovery"
+summary="Learn how to bootstrap and recover admin account.">
+
+== A temporary admin account
+
+A user or service admin account created using one of the methods described below is *temporary*. This means the account should exist only for the duration necessary to perform operations needed to gain permanent and more secure admin access. Various UI/UX elements, such as the Administration Console warning banner, labels, and log messages, will indicate to a {project_name} administrator that the account is temporary.
```

Comment:
Could you elaborate more on what temporary means here? Is the access available until the Keycloak node that was started with the parameters is alive?

Reply:
By "temporary" we mean "considered to be temporary by the user". Basically it's just semantical for now and only highlighted as a [warning](https://github.com/keycloak/keycloak/pull/31387) in the UI and logs. In the future, the accounts will be [expired](https://github.com/keycloak/keycloak/issues/30008#issuecomment-2151956072).

Reply:
I see. Should we mention in the docs they need to delete the temporary users manually?

Reply:
Addressed.

Reply:
Thank you @Pepo48